static/templates/confirm_dialog/confirm_deactivate_user.hbs

```diff
@@ -1,7 +1,29 @@
 <p>
     {{#tr}}
         When you deactivate <z-user></z-user>, they will be immediately logged out.
-        {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &lt;{{email}}&gt;{{/if}}{{/inline}}
+        {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &#40;{{email}}&#41;{{/if}}{{/inline}}
     {{/tr}}
 </p>
 <p>{{t "Their password will be cleared from our systems, and any bots they maintain will be disabled." }}</p>
+<label class="checkbox">
+    <input type="checkbox" class="send_email" data-key="{{ key }}" />
+    {{#tr}}
+    Send a notification email to <z-user></z-user>?
+    {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &#40;{{email}}&#41;{{/if}}{{/inline}}
+    {{/tr}}
+    <span></span>
+    {{> ../help_link_widget link="/help/deactivate-or-reactivate-a-user#notify-users-of-their-deactivation" }}
+</label>
+<div class="email_field">
+    <p class="border-top">
+        <strong>From:</strong> Zulip administrator
+        <br/>
```

Comment:
The actual "From" value will depend on the server's settings; let's just display the subject line here.